internal/ethapi/api.go

```diff
@@ -1856,12 +1856,13 @@ func (api *PublicDebugAPI) PrintBlock(ctx context.Context, number uint64) (strin
 }
 
 // SeedHash retrieves the seed hash of a block.
-func (api *PublicDebugAPI) SeedHash(ctx context.Context, number uint64) (string, error) {
+func (api *PublicDebugAPI) SeedHash(ctx context.Context, number uint64, epochLength uint64) (string, error) {
 	block, _ := api.b.BlockByNumber(ctx, rpc.BlockNumber(number))
 	if block == nil {
 		return "", fmt.Errorf("block #%d not found", number)
 	}
-	return fmt.Sprintf("0x%x", ethash.SeedHash(number)), nil
+	epoch := number / epochLength
```

Comment:
hmm, this will change the API signature which might brake stuff for people. I am not sure that we want that. Can we check if `api` exposes the config in order to check if Ecip1099 is enabled?

Reply:
Maybe using this `api.b.ChainConfig().GetEthashECIP1099Transition()`?